### ISSUE TITLE
Investigate testcafe integration

### ISSUE THREAD

**Issue by amilajack** [chentsulin/electron-react-boilerplate#1632]
https://github.com/DevExpress/testcafe

**Comment by doortts** [comment]
It's very interesting. What kind of testing are you thinking to use?

**Comment by amilajack** [comment]
I'm thinking of replacing `spectron` with `testcafe`. I need to investigate if that's possible. There seems to be [electron support for `testcafe`](https://github.com/DevExpress/testcafe-browser-provider-electron)

**Comment by goldylucks** [comment]
what about cypress? see [this solution](https://docs.cypress.io/faq/questions/using-cypress-faq.html#Can-I-test-my-Electron-app), they are using it to test their desktop app.

Another solution I'm not sure possible ..
Since erb uses webpack to serve the files, maybe it can expose a port for cypress to use?

**Comment by amilajack** [comment]
Yup, cypress is also something i want to investigate. I'll create an example for that as well

**Comment by amilajack** [comment]
Here's a link to [erb-testcafe-example](https://github.com/amilajack/erb-testcafe-example) btw
